src/features/runtime-checks/script-overload.js

```diff
@@ -1,5 +1,96 @@
 import { processAttr } from '../../utils.js'
 
+/**
+ * Indent a code block using braces
+ * @param {string} string
+ * @returns {string}
+ */
+function removeIndent (string) {
+    const lines = string.split('\n')
```

Comment:
This was much simpler than trying to make tagged template strings play ball, I spent an hour or so on trying to re-indent the values to the in place indent; it was a whole sea of sadness.

I'm pretty sure this will be faster perf wise and it appears to solve most use-cases.